### PR TITLE
Fix ubuntu signed by

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Fix regressions introduced by https://github.com/uyuni-project/uyuni/pull/9548

## Links

Belongs to: https://github.com/uyuni-project/uyuni/pull/9548
Port(s): 

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by CDellaGiusta — APPROVED** [review]
looks good

File: susemanager-utils/susemanager-sls/salt/channels/channels.repo [hunk 1]
```diff
@@ -23,6 +23,13 @@ Trusted: yes
 {%- endif %}
 {%- if args['gpgkeyurl'] is defined and args['gpgkeyurl'].startswith('file:///') %}
 Signed-By: {{ args['gpgkeyurl'][7:] }}
+{%- elif args['gpgkeyurl'] is not defined %}
+{#- bsc#1234251 #}
+{%- if grains['os'] == 'Debian' %}
+Signed-By: /usr/share/keyrings/debian-archive-keyring.gpg
+{%- else %}
+Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg
+{%- endif %}
```
[the remark this answers is not in the document]
Comment (mcalmer) [reply susemanager-utils/susemanager-sls/salt/channels/channels.repo:32]: I check now also for Ubuntu directly. If we have a debian OS which is neither Ubuntu nor Debian, Signed-by is not set.
This is ok as I have the feeling it is anyway the better way

[the remark this answers is not in the document]
Comment (raulillo82) [reply susemanager-utils/susemanager-sls/salt/channels/channels.repo:32]: As you already agreed here, and Carlo's approval went away with the resulting commit, I'll send my approval as a replacement.

**Review by raulillo82 — COMMENTED** [review]
Approving

**Review by raulillo82 — APPROVED** [review]

**Pull request merged**